### ISSUE TITLE
Return to Top Keyboard Focus

### ISSUE THREAD

**Issue by anna-zhang** [GSA/code-gov-front-end#454]
- [x] When "Return to Top" link is clicked, focus should be redirected to top of page skip-nav instead of the footer links
